src/game.cpp

```diff
@@ -1412,16 +1412,26 @@ bool game::do_turn()
 
     events.process();
     mission::process_all();
-    if( calendar::once_every( 1_days ) ) { // Midnight!
+
+    if( calendar::once_every( 1_days ) )
+    {
```

Comment:
This character should be on the line above, just like it is in the current code.